src/test/kotlin/com/terraformation/backend/nursery/db/BatchStoreTest.kt

```diff
@@ -164,6 +167,83 @@ internal class BatchStoreTest : DatabaseTest(), RunsAsUser {
     }
   }
 
+  @Nested
+  inner class GetSpeciesSummary {
+    @Test
+    fun `does not include germinating quantities in total withdrawn`() {
+      val batchId = insertBatch(speciesId = speciesId)
+      val withdrawalId = insertWithdrawal()
+      insertBatchWithdrawal(
+          batchId = batchId,
+          germinatingQuantityWithdrawn = 1,
+          notReadyQuantityWithdrawn = 2,
+          readyQuantityWithdrawn = 4,
+          withdrawalId = withdrawalId,
+      )
+
+      val summary = store.getSpeciesSummary(speciesId)
+
+      assertEquals(6, summary.totalWithdrawn, "Total withdrawn")
+    }
+
+    @Test
+    fun `does not include germinating quantities in loss rate`() {
+      val batchId =
+          insertBatch(
+              germinatingQuantity = 10,
+              notReadyQuantity = 1,
+              readyQuantity = 1,
+              speciesId = speciesId,
+          )
+      val withdrawalId = insertWithdrawal(purpose = WithdrawalPurpose.Dead)
+      insertBatchWithdrawal(
+          batchId = batchId,
+          germinatingQuantityWithdrawn = 20,
+          notReadyQuantityWithdrawn = 2,
+          readyQuantityWithdrawn = 3,
```

Comment:
I'm a bit confused on the logic here. It looks like the batch has 10/1/1 quantity to start with, but 20/2/3 is withdrawn. How does that work exactly? And below, wondering where the `7 total past + current seedlings` quantity comes from.

Reply:
The batch-level quantities are the current inventory. When we withdraw, we'll subtract the withdrawal amounts from them, though I haven't written that code yet. So the expected end state of a batch over time will be to have 0 for all three quantities (because all the plants have been withdrawn).

Reply:
Possibly of use as you look at these nursery PRs: https://terraformation.atlassian.net/wiki/spaces/SW/pages/648544257/Nursery+v1+data+model

Reply:
Thanks